.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,6 +11,11 @@ assignees: ''
 
 Check documentation: https://checkstyle.org/config_xxxxxx.html#NameOfAffectedCheck
 
+The cli helps in touching the roots of the problem better.
+It describes the problem better than 1,000 words.
+
+[Download and run cli]('https://checkstyle.org/cmdline.html#Download_and_Run')
```

Comment:
unfortunately this will be plain text paste to user, we try to avoid descriptions, and keep only text that user need to change.

Reply:
I think download and run cli link is very important to give in the first place for 2 reasons:
1. User may not have a latest checkstyle and could raise issue already solved. Now they could download the latest cli before creating an issue, and also get the help of how to run the cli from direct documentation.
2. User may not have used cli before, maybe he is using the editor's checkstyle. He will get to know about the cli and running it with terminal, and also could better understand the issue's bash template before modifying it.

Reply:
some idea on this point - https://github.com/checkstyle/checkstyle/pull/9122#discussion_r554487219